modules/abstract-utxo/src/abstractUtxoCoin.ts

```diff
@@ -670,14 +671,28 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
     return true;
   }
 
+  public async createKeySignatures(
+    prv: string,
+    backupKeychain: { pub: string },
+    bitgoKeychain: { pub: string }
+  ): Promise<{
+    backup: string;
+    bitgo: string;
+  }> {
+    return {
+      backup: (await this.signMessage({ prv }, backupKeychain.pub)).toString('hex'),
+      bitgo: (await this.signMessage({ prv }, bitgoKeychain.pub)).toString('hex'),
+    };
+  }
```

Comment:
move this to `baseCoin` and call it here: https://github.com/BitGo/BitGoJS/blob/529ad9e43e5123359c7417f3af2f5766e752474e/modules/sdk-core/src/bitgo/wallet/wallets.ts#L406

that way we have complementary `createKeySignatures`/`verifyKeySignatures`